create/remote-pipelines-tasks-resolvers.adoc

```diff
@@ -41,6 +41,15 @@ You can use the bundles resolver to specify a remote pipeline or task from a Tek
 include::modules/op-resolver-bundle-config.adoc[leveloffset=+2]
 include::modules/op-resolver-bundle.adoc[leveloffset=+2]
 
+[id="resolver-git_{context}"]
+== Specifying a remote pipeline or task from a Git repository
+
+You can use the Git resolver to specify a remote pipeline or task from a Git repostory. The repository must contain a YAML file that defines the pipeline or task. The Git resolver can access a repository either by cloning it anonymously or else by using the authenticated SCM API.
```

Comment:
```suggestion
You can use the Git resolver to specify a remote pipeline or task from a Git repostory. The repository must contain a YAML file that defines the pipeline or task. The Git resolver can access a repository either by cloning it anonymously or by using the authenticated SCM API.
```
Just saw this was moved from below, not critical, but I don't think "else" is needed